static/app/components/feedback/list/feedbackList.tsx

```diff
@@ -15,157 +8,94 @@ import ErrorBoundary from 'sentry/components/errorBoundary';
+        <InfiniteListState
+          queryResult={queryResult}
+          backgroundUpdatingMessage={() => null}
+          loadingMessage={() => <LoadingIndicator />}
         >
-          {({onRowsRendered, registerChild}) => (
-            <AutoSizer onResize={updateList}>
-              {({width, height}) => (
-                <ReactVirtualizedList
-                  deferredMeasurementCache={cache}
-                  height={height}
-                  noRowsRenderer={() =>
-                    isLoading ? (
-                      <LoadingIndicator />
-                    ) : (
-                      <NoFeedback
-                        title={t('Inbox Zero')}
-                        subtitle={t('You have two options: take a nap or be productive.')}
-                      />
-                    )
-                  }
-                  onRowsRendered={onRowsRendered}
-                  overscanRowCount={5}
-                  ref={e => {
-                    listRef.current = e;
-                    registerChild(e);
+          <InfiniteListItems<FeedbackIssueListItem>
+            estimateSize={() => 24}
+            queryResult={queryResult}
+            itemRenderer={({item}) => (
+              <ErrorBoundary mini>
+                <FeedbackListItem
+                  feedbackItem={item}
+                  isSelected={checkboxState.isSelected(item.id)}
+                  onSelect={() => {
+                    checkboxState.toggleSelected(item.id);
                   }}
-                  rowCount={issues.length}
-                  rowHeight={cache.rowHeight}
-                  rowRenderer={renderRow}
-                  width={width}
                 />
-              )}
-            </AutoSizer>
-          )}
-        </InfiniteLoader>
-        <FloatingContainer style={{top: '2px'}}>
-          {isFetchingPreviousPage ? (
-            <Tooltip title={t('Loading more feedback...')}>
-              <LoadingIndicator mini />
-            </Tooltip>
-          ) : null}
-        </FloatingContainer>
-        <FloatingContainer style={{bottom: '2px'}}>
-          {isFetchingNextPage ? (
-            <Tooltip title={t('Loading more feedback...')}>
-              <LoadingIndicator mini />
-            </Tooltip>
-          ) : null}
-        </FloatingContainer>
+              </ErrorBoundary>
+            )}
+            emptyMessage={() => <NoFeedback />}
+            loadingMoreMessage={() => (
+              <Centered>
+                <Tooltip title={t('Loading more feedback...')}>
+                  <LoadingIndicator mini />
+                </Tooltip>
+              </Centered>
+            )}
+            loadingCompleteMessage={() => null}
+          />
+        </InfiniteListState>
       </FeedbackListItems>
     </Fragment>
   );
 }
 
 const FeedbackListItems = styled('div')`
-  display: grid;
+  display: flex;
+  flex-direction: column;
   flex-grow: 1;
-  min-height: 300px;
+  padding-bottom: ${space(0.5)};
```

Comment:
This is to add spacing under the last item shown

<img width="460" height="111" alt="Image" src="https://github.com/user-attachments/assets/ca4c5ed6-e926-4a20-b536-b456bbcf0675" />